docs/maven-example.md

```diff
@@ -0,0 +1,56 @@
+# Fabric8 Maven Plugin + Kompose:
+
+Lets deploy a Springboot Java application with Docker Compose file using fabric8 maven plugin to Kubernetes/Openshift.
+
+__1. Clone the example project from github__
+```bash
+$ git clone https://github.com/piyush1594/Kompose-Example.git
+```
+Navigate (cd) to `Kompose-Example` directory.
+
+__2. Install the kompose through maven plugin__ 
+```
+$ mvn fabric8:install
+```
+This command installs the `kompose` and other binaries on the host.
+
+
+__3. Add the fabric8 maven plugin to project__ 
+```
+$ mvn io.fabric8:fabric8-maven-plugin:3.5.28:setup
+```
+This command adds the fabric8 maven plugin to `pom.xml` of project. 
```

Comment:
No need to start sentences with "This command".

You can simply say: "Add the Fabric8 maven plugin to the `pom.xml` of the project.

Speaking of `pom.xml`, as someone who has never used Maven, can you describe what this does? Need to elaborate on the instructions.